lib/pure/includes/oserr.nim

```diff
@@ -57,10 +57,10 @@ proc raiseOSError*(errorCode: OSErrorCode; additionalInfo = "") {.noinline.} =
   ## the message ``unknown OS error`` will be used.
   var e: ref OSError; new(e)
   e.errorCode = errorCode.int32
-  if additionalInfo.len == 0:
-    e.msg = osErrorMsg(errorCode)
-  else:
-    e.msg = osErrorMsg(errorCode) & "\nAdditional info: '" & additionalInfo & "'"
+  e.msg = osErrorMsg(errorCode)
+  if additionalInfo.len > 0:
+    e.msg.add  "; Additional info: "
+    e.msg.addQuoted additionalInfo
```

Comment:
Why are you changing this?

Reply:
`additionalInfo` could have quotes (eg from a cmd generated in startProcessAuxSpawn) so needs quoting otherwise msg will end up weird
i also removed the newline to avoid the weird formatting we had before: 
```
Error: no such file or directory
additional Info: /foo/bar [OsError] # the [OsError] on its own line with additional info was weird
```
=>
```
Error: no such file or directory; additional Info: /foo/bar [OsError]
```
disclaimer: i had added the `additional Info` extra param recently